lib/cfnvpn/templates/lambdas/auto_route_populator/quotas.py

```diff
@@ -0,0 +1,37 @@
+import boto3
+
+ROUTE_TABLE_QUOTA_CODE = 'L-401D78F7'
+AUTH_RULE_TABLE_QUOTA_CODE = 'L-9A1BC94B'
+EC2_SERVICE_CODE = 'ec2'
+IN_PROGRESS = ['PENDING', 'CASE_OPENED']
+
+def get_route_count(endpoint) -> int:
+  client = boto3.client('ec2')
+  response = client.describe_client_vpn_routes(
+    ClientVpnEndpointId=endpoint,
+  )
+  return len(response['Routes'])
+
+def quota_request_open(quota_code) -> bool:
+  client = boto3.client('service-quotas')
+  response = client.list_requested_service_quota_change_history_by_quota(
+    ServiceCode=EC2_SERVICE_CODE,
+    QuotaCode=quota_code
+  )
+  # Status='PENDING'|'CASE_OPENED'|'APPROVED'|'DENIED'|'CASE_CLOSED'
+  return True in (req['status'] in IN_PROGRESS for req in response['RequestedQuotas'])
```

Comment:
Is there a difference between this and any()?

Reply:
actually any should be used here, pretty sure this will run through all of the items in the generator where as any will short circuit when a match is found